programs/svm-spoke/src/instructions/admin.rs

```diff
@@ -247,4 +250,39 @@ pub fn relay_root_bundle(
     Ok(())
 }
 
-// TODO: add emergency_delete_root_bundle
+#[event_cpi]
+#[derive(Accounts)]
+#[instruction(root_bundle_id: u32)]
+pub struct EmergencyDeleteRootBundle<'info> {
+    #[account(
+        mut,
+        constraint = is_local_or_remote_owner(&signer, &state) @ CustomError::NotOwner
```

Comment:
We also need to add translation in handle_receive_message, so that we are able to invoke this from mainnet

Reply:
I'll add this together with remote call for relayRootBundle in separate PR